test/libgit2.jl

```diff
@@ -567,6 +570,178 @@ mktempdir() do dir
         @test creds.user == creds_user
         @test creds.pass == creds_pass
     #end
+
+    #@testset "SSH" begin
+        sshd_command = ""
+        ssh_repo = joinpath(dir, "Example.SSH")
+        if !is_windows()
+            try
+                sshd_command = strip(readstring(`which sshd`))
```

Comment:
That's not the purpose of this command. `sshd` refuses to be invoked as anything other than it's absolute path.